pkg/didmethod/peer/store.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peer
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"golang.org/x/xerrors"
```

Comment:
suggest to alias the import as errors (since it will be in 1.13+).

Reply:
Added alias for the xerrors import - https://github.com/hyperledger/aries-framework-go/compare/833a488e7fe5b8ef98198737e78a862440bb98a8..dcbe51e771012c337063cf3445d94e8d42b54c6b